src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,46 @@
 import React from 'react';
-
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface Props {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+
+export const MovieCard: React.FC<Props> = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+  imdbId,
+}) => (
+  <div className="card" key={imdbId} data-cy="card">
```

Comment:
The key always should be in map function call, [reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys](https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys)